exercises/D-logical-operators/exercise.js

```diff
@@ -11,14 +11,14 @@ let cssLevel = 4;
 
 // Finish the statement to check whether HTML, CSS knowledge are above 5
 // (hint: use the comparison operator from before)
-let htmlLevelAbove5;
-let cssLevelAbove5;
+let htmlLevelAbove5 = htmlLevel > 5;
+let cssLevelAbove5 = cssLevel > 5;
 
```

Comment:
Nice, you understand that logic operators can be used as a test to check whether statement values meet a certain condition. What could have been added to the code though? 
console.log("htmlLevelAbove5 greater than 5? : "+htmlLevelAbove5)
console.log("cssLevelAbove5 greater than 5? : "+cssLevelAbove5)
This way, we would know through the terminal whether the statements are true or false. Good job though